### ISSUE TITLE
Don't include Google Analytics tracking if GA_TRACKING_ID is unset

### ISSUE THREAD

**Issue by toolness** [18F/calc#868]
Oof, I should've noticed this in #855... We should just not include the GA `<script>` tag at all if `GA_TRACKING_ID` is unset/empty, rather than passing an empty string as the tracking ID into `ga()`, which is useless. This will allow folks who don't care about GA to easily opt-out of it (and our `common/ga` module will provide a no-op so that our code doesn't crash, too).

**Comment by jseppi** [comment]
Well, it isn't totally useless. It allows you to still see calls made (via the Google Analytics Debugger Chrome Extension, for instance), which is helpful for dev/debug.

**Comment by toolness** [comment]
Oh blerg.

It seems like there should be a way to disable it entirely, though. What happens if a "dummy" value is passed as the tracking ID? We could also distinguish between whether `GA_TRACKING_ID` is unset vs. the empty string--if it's unset, then we disable GA entirely, but if it's the empty string, we enable it and pass the empty string to `ga()`?

**Comment by jseppi** [comment]
If a dummy or blank value is passed, GA just doesn't have anywhere to send its data, but it still fires all its calls, which is nice.

Yeah, maybe just checking if it is unset is the best path. Then in dev/staging we could just set it to an empty string or some dummy value. I guess it is more overhead on us at the cost of making it better for downstream users of the codebase (though I don't even know if those users exist or how much we should be worrying about them).

**Comment by jseppi** [comment]
Alternatively, maybe we could just provide a `DISABLE_TRACKING_SCRIPTS` setting that will remove GA, ethnio, etc as a way to close #282 all the way?

**Comment by toolness** [comment]
Yeah, that's a good idea--honestly the main reason I filed that issue was because I was developing locally but I had a very slow internet connection, and the synchronous ethnio script was making the page loads take forever, so I wanted a way to disable it.

GA is already asynchronous, so it's not a big deal if it's just enabled all the time, and I just filed an issue for making DAP asynchronous in #887. If it's possible to load ...

wait, I just noticed our ethnio script has `async="true"`...

It must have been something else that was slowing down my page loads, I guess?

Whatever. I'm just gonna close this issue, it's not a big deal--if folks want to reuse our code and disable GA then we'll happily accept PRs!
